lib/tasks/valid_test_data_generator.rb

```diff
@@ -151,4 +153,80 @@ def random_or_nil_trn
       TRNGenerator.next
     end
   end
+
+  class NPQLeadProviderPopulater
+    class << self
+      def call(name:, total_schools: 10, participants_per_school: 10)
+        new(name: name).call(total_schools: total_schools, participants_per_school: participants_per_school)
+      end
+    end
+
+    def call(total_schools: 10, participants_per_school: 100)
```

Comment:
This `participants_per_school` variable is passed down through 4 different methods. We could just assign it to an instance variable to avoid that.